ci/scripts/tests/test_create_experiment.py

```diff
@@ -0,0 +1,22 @@
+from wxflow import Executable
+from shutil import rmtree
+import os
+
+_here = os.path.dirname(__file__)
+HOMEgfs = os.sep.join(_here.split(os.sep)[:-3])
+RUNDIR = os.path.join(_here, 'testdata/RUNDIR')
+case = "C48_ATM"
+
+
+def test_create_experiment():
+
+    env = os.environ.copy()
+    env['pslot'] = case
+    env['RUNTESTS'] = RUNDIR
+
+    create_experiment_script = Executable('../../../workflow/create_experiment.py')
+    create_experiment_script.add_default_arg(['-y', f'../../cases/pr/{case}.yaml', '--overwrite'])
```

Comment:
```suggestion
    create_experiment_script = Executable(f'{HOMEgfs}/workflow/create_experiment.py')
    create_experiment_script.add_default_arg(['-y', f'{HOMEgfs}/ci/cases/pr/{case}.yaml', '--overwrite'])
```
Use `HOMEgfs` declared on line 6 instead of relative paths